### DISCUSSION TITLE
Database Question

### DISCUSSION THREAD

**Discussion by Javi510348** [uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp#12] [Q&A] [answered]
I have a question regarding on the database. What team is going to be responsible for setting up the database for the project? 

If there is no group responsible for it, I could help with setting up the database.

**Comment by evnd04** [comment, marked as answer]
I would say each team is responsible for their databases at least for the data tables of each feature, for example, pet listings would probably need a pet database that stores the name, breed, age, etc, and the human users feature should use names, age, contact info and other related fields important to their particular feature,  the union of these would probably come in the second milestone or at the end of the first one. If you wish you could help any of the teams with their databases. If you have any suggestions on how to work the databases, languages, and any other technical aspect involved you could go to each team's discussion and leave a message.

**Reply by Javi510348** [reply]
Exactly. I think that we could have 1 database, but each group will be responsible of creating the table for their "group".  The only thing is that we need to make sure to be communicating with each other in case that there are elements that are going to be shared between tables.

I recommend using PostgreSQL since it's the one used in the Database course.  As for the hosting of the database, I don't have an option right now.  Any recommendation from anyone will be helpful because I think that having a database set up quickly will help us a lot down the road.

**Reply by evnd04** [reply]
I've never used PostgreSQL and haven't taken the Database course, but I've used MySQL which is probably very similar and at least that one is pretty simple and learner-friendly, but the developers decide the technicality so if anyone has a suggestion or thoughts on how the database should be created feel free to reply here. Also, feel free to open any discussions for additional communication between groups maybe for group leaders to maintain continuity throughout the features.

**Reply by Javi510348** [reply]
I am open to use MySQL. Let's see if someone else recommends something. Meanwhile, I will to some research about hosting the database. Thank You for answering my question.
